mim/commands/search.py

```diff
@@ -218,7 +219,8 @@ def load_metadata_from_local(package: str):
         version = get_installed_version(package)
         click.echo(f'local verison: {version}')
 
-        metadata_path = resource_filename(package, 'model_zoo.yml')
+        module_name = PKG2MODULE.get(package, package)
+        metadata_path = resource_filename(module_name, 'model_zoo.yml')
```

Comment:
Do we need to support both model_zoo.yml and model-index.yml?

Reply:
Currently, we should support both of them because we are renaming the model_zoo.yml to model-index.yml